File: JavaPOP/src/test/TestSelenium.java
package test;

import org.junit.Before;
import org.junit.Test;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import pages.MainPage;

public class TestSelenium {

        WebDriver driver;

        //Page defined
        MainPage firstPage;

        @Before
        public void setup() {
            System.setProperty("webdriver.gecko.driver", ".\\src\\lib\\geckodriver.exe");
            driver = new FirefoxDriver();
            driver.get("https://www.google.com/");

            //Page initialized
            firstPage = new MainPage(driver);
        }

        @Test
        public void searchGoogle()
        {
            firstPage.findQAdevPage();
            firstPage.chooseQAdevPage();
            firstPage.close();
        }
}

File: C#POP/SeleniumTest/SeleniumTest/Test/TestSelenium.cs
using System;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using OpenQA.Selenium;
using OpenQA.Selenium.Firefox;
using SeleniumTest.PageObject;
using System.IO;

namespace SeleniumTest
{
    [TestClass]
    public class TestSelenium
    {
        private IWebDriver driver;
        private MainPage firstPage;

        [TestInitialize]
        public void setup()
        {
            FirefoxDriverService service = FirefoxDriverService.CreateDefaultService(@"\lib\");
            FirefoxOptions options = new FirefoxOptions();
            TimeSpan time = TimeSpan.FromSeconds(10);
            driver = new FirefoxDriver(service, options, time);

            driver.Url = "https://www.google.com/";

            //Page initialized
            firstPage = new MainPage(driver);
        }

        [TestMethod]
        public void SearchGoogleTest()
        {
            firstPage.findQAdevPage();
            firstPage.chooseQAdevPage();
            firstPage.close();
        }
    }
}
